iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Network.java

```diff
@@ -99,14 +99,32 @@
  */
 public interface Network extends Container<Network> {
 
+    String SUBNETWORKS_ARE_UNSUPPORTED = "This network doesn't support subnetworks.";
```

Comment:
Note for reviewers: in English the good term is `subnetwork` so we are going in another commit all `subNetwork` to `subnetwork`. But it does not change the design of the API presented in that PR.